tests/containers/rootless_podman.pm

```diff
@@ -76,10 +76,7 @@ sub run {
         validate_script_output "id $testapi::username", sub { /systemd-journal/ };
     }
 
-    if ((is_s390x || is_ppc64le) && check_bsc1192051()) {
-        record_soft_failure("bsc#1192051 - Permission denied for faccessat2");
-        return;
-    }
+    check_bsc1192051();
```

Comment:
Wait, this alters the code flow as well.

I would much rather have the `record_soft_failure` here turned into a `die` and leave the check as it is. Here you are not checking the return value of the checking routine, and therefore effectively removing this regression check.

Reply:
I guess s/script_run/assert_script_run/ would be just fine

Reply:
Yes, that would also work.

Reply:
Fixed.